docs/index.bs

```diff
@@ -207,6 +223,8 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
     A [=/service worker=] has an associated <dfn>all fetch listeners are empty flag</dfn>. It is initially unset.
 
+    A [=/service worker=] has an associated <dfn>static router rules object</dfn>. It is initially unset.
```

Comment:
Can you state its type? Per the below, I think it should be a list of router rules. (Which makes the use of the "object" suffix kind of strange.)

Reply:
renamed to "list of router rules".